app/lib/glean.rb

```diff
@@ -0,0 +1,178 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# AUTOGENERATED BY glean_parser v0.1.dev1004+g4822435. DO NOT EDIT.
+
+# frozen_string_literal: true
+
+# requires json, securerandom, and logger libs
+require 'json'
+require 'securerandom'
+require 'logger'
+require 'rbconfig'
+
+# this will be used for identifying logs that need to forward to Moz Data Pipeline
+GLEAN_EVENT_MOZLOG_TYPE = 'glean-server-event'
+
+# helper module to get OS level info for logs
+module OS
```

Comment:
I missed this in https://github.com/mozilla/glean_parser/pull/620 but I'd say the OS is not necessary in this telemetry collection.
In principle, we're aiming for Glean telemetry on the server to be used to collect user-level behavioral data. Server os name/version is more of a service-level operational data that we didn't want to mix in. Because these fields are required in the ping schema we are [setting them to `Unknown` in javascript template](https://github.com/mozilla/glean_parser/blob/e1adee5df11591438a5199438e0a98df02cb7e23/glean_parser/templates/javascript_server.jinja2#L115-L120).

Reply:
cc @braunreyes

Reply:
cool...I will get rid of it

Reply:
removed the unneeded OS module